app/models/xccdf/benchmark_searching.rb

```diff
@@ -0,0 +1,107 @@
+# frozen_string_literal: true
```

Comment:
I think the right location for this file should be:
```
app/models/concerns/xccdf/benchmark_searching.rb
```
Or maybe:
```
app/models/xccdf/concerns/benchmark_searching.rb
```
But I think the first one is the more rails-y

Reply:
The latter did not worked. The former can do it maybe...

Reply:
I'm ok to have this outside of the Xccdf module. One day, we'll probably remove benchmark.rb from Xccdf as well. It was intended to show our models that have a 1 to 1 counterpart in the SCAP model, but we never actually followed up with it for rules, profiles, test results, etc., and I don't really see why it's useful anyway. To use it within benchmark.rb, all we'd need is the root namespace, `include ::BenchmarkSearching`.

Reply:
Moved to top model-level concern.